api/controllers/login.js

```diff
@@ -16,6 +16,10 @@ module.exports = function(store) {
 
 			return res.success(token);
 		} catch (e) {
+			if (e.Error === 'LockError') {
+				res.set('Retry-After', e.retry);
```

Comment:
Please format the retry date as an [ISO 8601 string](http://momentjs.com/docs/#/displaying/as-iso-string/), if that is not already the case.